src/components/Popups/PopupReply.tsx

```diff
@@ -36,34 +38,28 @@ export default function PopupReply({
     };
     const handleCancel = (event: React.MouseEvent): void => {
         handleEvent(event);
-        onCancel(text);
+        onCancel();
     };
-    const handleChange = (event: React.ChangeEvent<HTMLTextAreaElement>): void => {
-        setText(event.target.value);
-        onChange(event.target.value);
+    const handleChange = (nextEditorState: EditorState): void => {
+        onChange(nextEditorState);
     };
     const handleSubmit = (event: React.FormEvent): void => {
         event.preventDefault();
         handleEvent(event);
-        onSubmit(text);
+        onSubmit();
```

Comment:
Text string should still be passed up to avoid the component making assumptions about how its parent behaves.